requirements.txt

```diff
@@ -9,7 +9,7 @@ cffi==1.10.0
 chardet==3.0.4
 click==6.7
 configparser==3.5.0
-cryptography==1.7.2
+cryptography==2.3.1
```

Comment:
It would be nice to see a test (or just a reasoned argument) that confirmed this change didn't break the app

Reply:
Also, linking in #22